src/app/zap-templates/zcl/data-model/chip/content-launch-cluster.xml

```diff
@@ -57,33 +58,35 @@ limitations under the License.
 
   <struct name="AdditionalInfoStruct">
     <cluster code="0x050a"/>
-    <item name="Name" type="char_string"/>
-    <item name="Value" type="char_string"/>
+    <item name="Name" type="char_string" length="256" />
+    <!-- TODO: spec sets this max size, but seems impossible given UDP size limits -->
```

Comment:
This is 1.3-proposed stuff....  I think they mean to only support it over TCP, and it might not stay in this form.

Reply:
The size seems to be in 1.2.